docs/quickstart.rst

```diff
@@ -0,0 +1,40 @@
+.. _quickstart:
+
+Quickstart
+==========
+
+Create a `SQLAlchemy <http://www.sqlalchemy.org/>`__ model and a
```

Comment:
Nit: since there's not much narrative documentation yet, this can probably just go in index.rst.